lib/elements/satin_column.py

```diff
@@ -99,6 +99,24 @@ def e_stitch(self):
     def max_stitch_length(self):
         return self.get_float_param("max_stitch_length_mm") or None
 
+    @property
+    @param('short_stitch_inset',
+           _('Short stitch inset'),
+           tooltip=_('Maximum stitch length for split stitches.'),
+           type='float', unit="%",
+           default=10)
+    def short_stitch_inset(self):
+        return self.get_float_param("short_stitch_inset", 10) / 100
+
+    @property
+    @param('short_stitch_distance',
+           _('Short stitch distance'),
+           tooltip=_('Do short stitches if the distance between stitches is smaller than this.'),
+           type='float', unit="mm",
+           default=0.25)
+    def short_stitch_distance(self):
+        return self.get_float_param("short_stitch_distance", 0.25) * PIXELS_PER_MM
```

Comment:
If you name the param `short_stitch_distance_mm` (on this line and on 112) then you don't have to multiply by PIXELS_PER_MM.  `get_float_param()` will do it for you.  You can still use `def short_stitch_distance(self):`, you don't have to rename the function too.